Jellyfin.Plugin.Streamyfin/Configuration/Config.cs

```diff
@@ -30,6 +30,7 @@ public class Config
   //public Bla Wit { get; set; }
   public Search? marlinSearch { get; set; }
```

Comment:
Lets rework marlin search to match the setting in Streamyfin and add it under the settings with lock.